packages/cli/src/ui/contexts/KeypressContext.tsx

```diff
@@ -304,6 +311,19 @@ export function KeypressProvider({
       if (key.name === 'return' && key.sequence === `${ESC}\r`) {
         key.meta = true;
       }
+
+      // readline doesn't set the ctrl flag for byte sequences x7f and x08
```

Comment:
this logic looks like it runs even if the terminal uses kitty. Can you make it more explicit that this logic is only applied if the terminal is not kitty if htat is your intend?